src/Glimpse.Common/Broker/MessageContext.cs

```diff
@@ -9,6 +9,7 @@ public class MessageContext
 
         public string Type { get; set; }
 
+        [JsonIgnore]
```

Comment:
Good pickup... I'm trying to find a better place to put this. Not sure if should be here.